Dockerfile

```diff
@@ -75,3 +75,10 @@ RUN chmod u-w sites/default/settings.php \
 RUN chown -R www-data:www-data ./
 
 USER www-data
+
+FROM build as local
+USER root
+RUN pecl install xdebug-2.9.8 \
+  && docker-php-ext-enable xdebug
+USER www-data
+
```

Comment:
This looks good! The default build artefact this way is a local dev build, which it’s safer to make explicit. 

If we add a single line to the end of this file like
‘’’
FROM build as production
‘’’

That should do it